src/explore-education-statistics-frontend/src/modules/data-catalogue/components/DataSetFileApiQuickStart.tsx

```diff
@@ -12,10 +14,37 @@ interface Props {
 
 export default function DataSetFileApiQuickStart({ id, name, version }: Props) {
   return (
-    <DataSetFilePageSection
-      heading={pageSections.apiQuickStart}
-      id="apiQuickStart"
-    >
+    <DataSetFilePageSection heading={pageSections.api} id="api">
+      <ChevronGrid>
+        <ChevronCard
+          cardSize="l"
+          description="This data set is available via an API, follow the link to get all the information to get started. 
+          The documentation provides full guidance and examples on how to make the most from our data sets."
+          headingSize="m"
+          link={
+            <Link to={process.env.PUBLIC_API_DOCS_URL ?? ''}>
```

Comment:
I'm just wondering when we ever have the `PUBLIC_API_DOCS_URL` variable undefined.  Do we handle all environment variables this way, with explicit handling of undefined values?

Reply:
Good spot, that check is unnecessary so I've removed it.